docs/endpoints/posts.md

```diff
@@ -139,6 +139,9 @@ POST /api/v2/posts
     `Y-m-d H:i:s` format. When the post was created.
   - **updated_at**: (string) optional.
     `Y-m-d H:i:s` format. When the post was last updated.
+  - **type**: (string).
+    The type of post submitted e.g. photo, voter-reg, text
+  - **action**: (string).
```

Comment:
We define this as: `Describes the bucket the action is tied to. A campaign could ask for multiple types of actions throughout the life of the campaign.` in the DB

Reply:
ah ok that is great! I'll add to the documentation :)